script/version.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2020 Colin B. Macdonald
+# SPDX-License-Identifier: FSFAP
```

Comment:
@rbeezer I'm old fashioned and usually try for per-file copyright notices.  For a trivial file like this I typically use the "FSF All Permissive" which is pretty much public domain.

If you don't care about any of this, you can take my name off, or remove the copyright notice altogether --- it is afterall "all permissive" :-)